make-money/break-moment.md

```diff
@@ -0,0 +1,27 @@
+# Break Moment
+
+Breaks provide custom brand messages during natural pauses in application use.  Since users expect pauses at certain points in your application, displaying ads at this time will not take away from the user experience. There are a variety of ads that can be displayed during a break moments including; rich media,video, and app download ads.
+
+## Break Moment Examples
+
+### Games
+
+* Player completes level (win or loss)
+* Player is waiting for leaderboard to load
+* Player reaches loading screen
+
+### Lifestyle Apps
+
+* User switches tasks
+* User schanges window view
+* User reaches loading screen
+
+
+## Moment Messaging
+
+Break moments occur during natural pauses in your application, therefore you do not need to explain why a user is seeing an ad. The goal is to allow players to quickly interact with a break ad unit so they can get back into the action as quickly as possible. 
```

Comment:
The verb 'quickly' is used twice - either swap it with another, or maybe drop the first one?